src/shared/components/dataAccessTokens/DataAccessTokensDropdown.tsx

```diff
@@ -53,7 +53,10 @@ export class DataAccessTokensDropdown extends React.Component<
                 action: (
                     <a
                         href={buildCBioPortalPageUrl(
-                            this.props.appStore.logoutUrl
+                            this.props.appStore.logoutUrl,
+                            {
+                                local: AppConfig.serverConfig.saml_logout_local.toString(),
```

Comment:
@amirineni1 @alisman From what I understand, if saml_logout_local is true, the logout URL is suppose to be extended to something like the following (as opposed to "/saml/logout"):

> /saml/logout?local=true

Where does this happen?  Whats happens when you convert a boolean to a string here?

Reply:
@n1zea144 src/appShell/App/PortalHeader.tsx is the file will consume DataAccessTokensDropdown to populate "Sign Out" link.  Since parameter "local" is a type of "string", converted boolean value to string. This would match the type and will not show any typescript errors. toString method simply converts a boolean value to string and return either "true" or "false" value.

Reply:
@n1zea144 could you please review and merge these changes? Let me know if any additional questions. thanks

Reply:
Sorry I'm being so "thick" here - as I understand it, default logout URL is something like this:

"/saml/logout", but when saml_logout_local is "true" the URL should change to something like this:

"/saml/logout?local=true"

where in PortalHeader.tsx does this happen?

Reply:
@n1zea144 
PortalHeader.tsx below code calls DataAccessTokensDropdown
`                                        <Dropdown.Menu
                                            style={{
                                                paddingLeft: 10,
                                                overflow: 'auto',
                                                maxHeight: 300,
                                                whiteSpace: 'nowrap',
                                            }}
                                        >
                                            <DataAccessTokensDropdown
                                                appStore={this.props.appStore}
                                            />
                                        </Dropdown.Menu>`

In DataAccessTokensDropdown.tsx signout URL will be constructed using buildCBioPortalPageUrl method. This method requires two parameters 
- one is the actual URL that should return by this.props.appStore.logoutUrl method ("/saml/logout")
- second one is query parameter (local=true|false) that should be retrived from the serverConfig
`
               id: 'signout',
                action: (
                    <a
                        href={buildCBioPortalPageUrl(
                            this.props.appStore.logoutUrl, ---> "/saml/logout"
                            {
                                local: AppConfig.serverConfig.saml_logout_local.toString(), ---> "local=true|false"
                            }
                        )}
                    >
`

buildCBioPortalPageUrl("/saml/logout", {local:true})

Reply:
I see, thats for the follow-up info.